client/my-sites/plan-price/index.tsx

```diff
@@ -218,6 +218,16 @@ function PriceWithoutHtml( {
 	return <>{ priceObj.integer }</>;
 }
 
+export function PriceAsString( price: number, currencyCode: string, isSmallestUnit = false ) {
```

Comment:
🤔 Not a huge deal, but is this supposed to be a React component, or a helper function? Typically React components are capitalized like this is, but the rest of it doesn't follow the rules of a functional component: it has multiple parameters instead of a single `props` parameter and it's being called inline at its call site rather than rendered with JSX.

I'd suggest picking one or the other.

Reply:
I intended it as a helper function - since this PR is fixing some pricing display text, we can flag this to clean up